flink-runtime/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcActor.java

```diff
@@ -488,16 +493,16 @@ default String invalidStateTransitionMessage(State targetState) {
 
 	@SuppressWarnings("Singleton")
 	enum StartedState implements State {
-		INSTANCE;
+		STARTED;
```

Comment:
This looks like a cosmetical change and it is not covered by the commit message.

Reply:
True, I will update the commit message.